src/coreclr/init-distro-rid.sh

```diff
@@ -154,7 +154,7 @@ initDistroRidGlobal()
         local distroRid=""
 
         # Check for musl-based distros (e.g Alpine Linux, Void Linux).
-        if (ldd --version 2>&1 || true) | grep -q musl; then
+        if "${rootfsDir}/usr/bin/ldd" --version 2>&1 | grep -q musl; then
```

Comment:
Do we need to worry about machines that have `ldd` in `PATH` somewhere other than `/usr/bin/ldd`? (I'm totally not an expert here.)

Reply:
It is possible to customize the environment and not even have `os-release` under /etc. But we have other assumptions based on the paths set in stock settings.

On musl-based distros (Alpine and Void), ldd is present at /usr/bin, so I think it is ok and it is unusual someone don’t have ldd there. Also, the base-minimal image of alpine (5MB in size) also include it and their own packages depend on it.

Reply:
Sounds good to me, thanks for looking into it!